AISKU/src/Initialization.ts

```diff
@@ -213,12 +210,10 @@ export class Initialization implements IApplicationInsights {
     /**
      * Log a dependency call (e.g. ajax)
      * @param {IDependencyTelemetry} dependency
-     * @param {{[key: string]: any}} [customProperties]
-     * @param {{[key: string]: any}} [systemProperties]
      * @memberof Initialization
      */
-    public trackDependencyData(dependency: IDependencyTelemetry, customProperties?: {[key: string]: any}): void {
-        this.dependencies.trackDependencyData(dependency, customProperties);
+    public trackDependencyData(dependency: IDependencyTelemetry): void {
```

Comment:
So how custom properties will be send now?, dependency.properties?

Reply:
Yes, but this is only the external facing API. No change to any internal dependencies plugin APIs.